### DISCUSSION TITLE
No 'play' option with huskcontainers

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15100] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You don't get a play option with husk containers, thus have no significant way to increase their happiness. 

### Reproduction steps

[huskcontainertest.zip](https://github.com/user-attachments/files/17690690/huskcontainertest.zip)

1. Load save file 
2. Give yourself 100 huskcult reputation
3. Buy huskcontainer and try playing with it

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
literally unplayable

**Comment by Regalis11** [comment]
Fixed in the latest unstable build, and the fix will be included in the next release.
